fotoapparat/src/main/java/io/fotoapparat/hardware/v1/CameraParametersDecorator.java

```diff
@@ -128,6 +132,14 @@ public void setPreviewSize(int width, int height) {
         cameraParameters.setPreviewSize(width, height);
     }
 
+    public int getMaxNumFocusAreas() {
+        return cameraParameters.getMaxNumFocusAreas();
+    }
+
+    public int getMaxNumMeteringAreas() {
```

Comment:
Avoid shortcuts in the names. I would either call it `getMaxNumberOfMeteringAreas` or just `getMaxMeteringAreas`. Both are unambiguous, so I would go for a shorter one.